PWG/EMCAL/EMCALtasks/AliEmcalCorrectionClusterNonLinearity.h

```diff
@@ -43,6 +43,8 @@ class AliEmcalCorrectionClusterNonLinearity : public AliEmcalCorrectionComponent
   TH1F                  *fEnergyDistAfter;           //!<!energy distribution after
   TH2F                  *fEnergyTimeHistAfter;       //!<!energy/time distribution after
   
+  Bool_t                 fSetClusterE;               ///< Stored it in cluster->E() and not cluster->GetNonLinCorrEnergy()
```

Comment:
Note that this comment isn't quite right, because we still set the cluster non-linearity energy regardless of the setting.

Anyway, in conjunction with the rename in the YAML config, I would suggest that this should be `fForceSetClusterE` and the comment should be along the lines of

```
// Only for backwards compatibility, force cluster->E() to be set to the cluster non-linearity corrected energy. Off by default. For the standard methods, see: http://alidoc.cern.ch/AliPhysics/master/READMEcontainers.html#emcalContainerClusterEnergyCorrections
```